user/views.py

```diff
@@ -65,13 +66,128 @@ def post(self, request):
+                posted_review             = already_posted_review.get()
+                posted_review.score       = review_rating
+                posted_review.comment     = review_text
+                posted_review.review_type = review_type_info
+                posted_review.save()
+
+            else: # 리뷰등록
+                Review(
+                    score       = review_rating,
+                    comment     = review_text,
+                    review_type = review_type_info,
+                    film        = film_info,
+                    user        = user_info
+                ).save()
+
+            # 해당 영화의 평균별점 재계산
+            review_set        = Review.objects.filter(film = film_info)
+            review_set_number = review_set.count()
+            total_rating      = 0
+
+            for review_element in review_set:
+                total_rating += review_element.score
+
+            film_info.avg_rating = total_rating / review_set_number
+            film_info.save()
+
+            return JsonResponse({"message":"POST_REVIEW_SUCCESS"}, status=200)
+
+        except JSONDecodeError:
+            return JsonResponse({"message":"JSONDecodeError"}, status=400)
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+        except Review.DoesNotExist:
+            return JsonResponse({"message":"NOT_EXISTS_REVIEW"}, status=400)
+
+
+    @token_authorization
+    def delete(self,request): # 리뷰 삭제
+        try:
+            data      = json.loads(request.body)
+            film_id   = data['film_id']
+            film_info = Film.objects.get(id=film_id)
+            user_info = request.user            
+
+            if not user_info: # 유저 정보가 없는 경우
+                return JsonResponse({"message": "INVALIDE_USER"}, status=400) 
+
+            Review.objects.get(film = film_info, user = user_info).delete()
+            
+            # 해당 영화의 평균별점 재계산
+            review_set        = Review.objects.filter(film = film_info)
+            review_set_number = review_set.count()
+            total_rating      = 0
+
+            for review_element in review_set:
+                total_rating += review_element.score
+            
+            if review_set_number == 0:
+                film_info.avg_rating = 0
+            else:
+                film_info.avg_rating = total_rating / review_set_number
+
+            film_info.save()        
+
+            return JsonResponse({"message": "DELETE_REVIEW_SUCCESS"}, status=200)
+
+        except JSONDecodeError:
+            return JsonResponse({"message":"JSONDecodeError"}, status=401)
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+        except Review.DoesNotExist:
+            return JsonResponse({"message":"NOT_EXISTS_REVIEW"}, status=401)
+
+class ReviewCount(View):
+    def get(self, request):
+        review_count = Review.objects.all().count()
```

Comment:
```suggestion
        review_count = Review.objects.count()
```
all() 없이도 사용하실 수 있습니다